README.md

```diff
@@ -0,0 +1,164 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Expense Insight ](#-expense-insight-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [🚀 Description Video](#-description-video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Expense Insight <a name="about-project"></a>
+Expense Insight is a web application built with Ruby on Rails that helps users manage their expenses by tracking transactions and categorizing spending.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li>
+      <a href="https://www.ruby-lang.org/en/">Ruby</a>
+    </li>
+     <li>
+      <a href="https://rubyonrails.org/">Ruby on rails </a>
+    </li>
+     <li>
+      <a href="https://www.postgresql.org/">PostgresQL</a>
+    </li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- User Authentication: Register and log in securely to keep your expense data private.
+- Category Management: Create spending categories.
+- Transaction Tracking: Log and view your transactions, associating them with specific categories.
+- Expense Summaries: See the total amount spent in each category.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Visit the live project](https://expense-insight.onrender.com/)
+
+## 🚀 Description Video<a name="video-demo"></a>
+
+- [Link to the description video](https://www.loom.com/share/6ed3729c0f954284adaf0e3478426e4e)
+
```

Comment:
- [ ] Great job working on the required changes but I think it would be great if you update your video presentation with a new video mentioning a piece of code you are proud of, However as required for the project requirements, please make sure that your video also contains a highlight about a piece of the code you find interesting or you are proud of making.✅⭐💯🚀